cScripts/CavFnc/functions/modules/fn_moduleCreateTimer.sqf

```diff
@@ -0,0 +1,52 @@
+#include "..\script_component.hpp";
+/*
+ * Author: Sweetwater.I
+ * This module function creates a countdown timer
+ * A Heavily bastardized copy of the zeus_curator module for bis_fnc_countdown
+ *
+ * Arguments:
+ * 0: None
+ *
+ * Example:
+ * this call cScripts_fnc_moduleCreateTimer
+ *
+ * Public: No
+ */
+params ["_Ctimer", "_value", "_valueText", "_time"];
+
+private _dialogResult = [
+    "Countdown Timer",
+    [["Seconds","", "60"]]
+] call Ares_fnc_showChooseDialog;
+
+if (count _dialogResult == 0) exitWith {};
+    _Ctimer = _dialogResult select 0;
+
+_time = [] call bis_fnc_countdown;
+
+if (_time <= 0) then {_time = estimatedendservertime - servertime;};
+_value = _time call compile _Ctimer;
+_value = _value param [0,-1,[0]];
+
+if (_value >= 0) then {
+    if (abs (([] call bis_fnc_missionTimeLeft) - _value) > 60) then {
+    _valueText = format ["%1 %2",ceil (_value / 60),localize "STR_A3_RSCMPPROGRESS_min"];
+    [["Countdown",[_valueText]],"bis_fnc_showNotification"] call bis_fnc_mp;
+};
+            
+if (ismultiplayer) then {
+    estimatedtimeleft _value;
+} else {
+    _value call bis_fnc_countdown;
+    };
+} else {
```

Comment:
this is weirdly set up and will cause errors

Reply:
this in particular is not due to indent its due to the setup of the if sats.
For default structure of if sats here for used here is the following format:
```
if (_condition) then {
    //Code
} else {
    //Code
};
if (_condition) then { /*code*/};
```

But above you have done like this witch make no sence:
```
if (ismultiplayer) then {
    estimatedtimeleft _value;
} else {
    _value call bis_fnc_countdown;
    };
} else {
if (ismultiplayer) then {
    estimatedtimeleft 1e10;
} else {
    -1 call bis_fnc_countdown;
};
```
line 42
starts with `}; } else {`

Reply:
think I see what happend here now it's the indent. I run like this:
```
if () then {
    if () then {
        if () then {
        } 
    } else {
    };
} ;

```